diff --git a/lib/core/GraphQLHandler-C5CUIS_N.d.ts b/lib/core/GraphQLHandler-C5CUIS_N.d.ts
index 7a0d0177142099f7f1de293eaed7015375f30af5..aef425b35d5dce4cbf5587c5c9d3adc1fe60be7d 100644
--- a/lib/core/GraphQLHandler-C5CUIS_N.d.ts
+++ b/lib/core/GraphQLHandler-C5CUIS_N.d.ts
@@ -27,7 +27,7 @@ declare function parseGraphQLRequest(request: Request): Promise<ParsedGraphQLReq
 
 type ExpectedOperationTypeNode = OperationTypeNode | 'all';
 type GraphQLHandlerNameSelector = DocumentNode | RegExp | string;
-type GraphQLQuery = Record<string, any>;
+type GraphQLQuery = Record<string, any> | null;
 type GraphQLVariables = Record<string, any>;
 interface GraphQLHandlerInfo extends RequestHandlerDefaultInfo {
     operationType: ExpectedOperationTypeNode;
